app/controllers/projects_controller.rb

```diff
@@ -305,7 +304,7 @@ def fork
     child = Project.new name: @project.name,
                         user_id: current_user.id,
                         uniqueurl: @project.uniqueurl,
-                        parent: @project.parent
+                        parent_id: @project.id
```

Comment:
The problem with this, and I mentioned it in my comment to #229, is that if you're forking a project that is _itself_ a fork, the parent of the new project should be the original project, not the fork. Setting a non-forked project as its own parent, and using the parent id as the parent of forks accomplished this.